config.mak

```diff
@@ -131,6 +134,10 @@ else
 
 endif
 
+ifeq ($(CODE_COVERAGE),true)
+    SGXLKL_CFLAGS_ENCLAVE_EXTRA += -fprofile-arcs -ftest-coverage
```

Comment:
Should we put right hand side in quotes?

Reply:
It works without quotes, and that's consistent with other flags in the same file.